fence/agents/pve/fence_pve.py

```diff
@@ -179,6 +179,12 @@ def main():
 	if options["auth"] is None:
 		fail(EC_LOGIN_DENIED)
 
+	# Workaround for unsupported API call on some Proxmox hosts
+	outlets = get_outlet_list(None, options)        # Unsupported API-Call will result in value: None
+	if outlets is None:
+		result = fence_action(None, options, set_power_status, get_power_status, None)
+		sys.exit(result)
+
```

Comment:
Looks good, but you can skip the if-section and change to outlets instead of get_outlet_list in the result-line below instead.